index.js

```diff
@@ -32,10 +34,38 @@ const { EthicalAds } = require('./ethicalAds')
   const url = new Url({ config, docs })
   const github = new GitHub({ config })
   const ethicalAds = new EthicalAds({ config, docs })
+  const dodTopLevelDependencyRetriever = new DodTopLevelDependencyRetriever({ db, config, s3, sqs })
 
   await db.setup()
   stripe.init()
 
+  /** Distribute Org Donations poller */
+  const dodPoller = Consumer.create({
+    queueUrl: config.getDistributeOrgDonationQueueUrl(),
```

Comment:
is this the correct one? i thought it would be the org-github-retriever queue (but maybe that is what this fn returns)